configure.ac

```diff
@@ -826,7 +830,13 @@ fi
 
 # libpcap can require libnl
 AC_SEARCH_LIBS([nl_handle_alloc], [nl],
-        [AC_MSG_NOTICE([Unable to find nl library - may be needed by libpcap])])
+        [AC_MSG_NOTICE([Unable to find xdp library - may be needed by libpcap])])
```

Comment:
is this a copy/paste error?

Reply:
Yes, its very likely copy-paste mistake/quick experiment left in the final PR.
This line should be unmodified, as in the original code.